src/qt/intro.h

```diff
@@ -69,10 +69,12 @@ private Q_SLOTS:
     QString pathToCheck;
     uint64_t m_blockchain_size;
     uint64_t m_chain_state_size;
+    bool pruneWasNotSet;
 
     void startThread();
     void checkPath(const QString &dataDir);
     QString getPathToCheck();
+    void setSizeWarningLabel(bool is_pruned);
```

Comment:
`ui->prune->isChecked()` is accessible here. `bool is_pruned` seems redundant.

Reply:
Oh yeah, it is just a remaining of the first version of this patch